plugins/systemd/systemd_status

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/python3 -tt
+# -*- coding: utf-8 -*-
```

Comment:
I think, this is a `python2` marker?

Reply:
python3's default is utf-8, so above "coding" marker just sets it again to utf-8 = does nothing. So it could be omitted. It (and -tt) is mostly leftover from quick py2->py3 conversion. 

Some older editors still require it as they default to py2 and it's charset. I can take it away as my $EDITOR doesn't need it.

Reply:
The -*- tag is for the benefit of Emacs I believe.